recipe/recipe.yaml

```diff
@@ -44,12 +44,12 @@ outputs:
           then:
             - openssl
       run:
-        - python
+        - python >=3.9
```

Comment:
Shouldn't it be python_min here?

Reply:
let's quickly see if the issue is fixed and then i can fiddle with it some more

Reply:
fixed